### PR TITLE
added 3 favicons, asset folder, & link to favicon

### PR DESCRIPTION
**Pull request by JS-goose**
Added assets folder to house icons
Added 3 possible favicons from Material Design
Added attribution for icons
Added icon link that meets W3 standards https://www.w3.org/2005/10/howto-favicon

### CODE REVIEW

**Comment by pavi2410** [comment]
Can you please the favicon to this one, please?
https://github.com/pavi2410/URL-Inspector/issues/5#issuecomment-546713318

**Comment by pavi2410** [comment]
Also, why there's 2 unused images?

**Comment by JS-goose** [comment]
I added the other images to give you choices as I wasn't sure which icon you would like.

I have changed it to the detective icon you referenced 👍

**Comment by pavi2410** [comment]
htmlpreview.github.io/?https://github.com/JS-goose/URL-Inspector/blob/00ecae13c4379b89fbc6deb9a141808cf6afa8c3/index.html

File: index.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="theme-color" content="#eceff1">
     <link rel="stylesheet" type="text/css" href="./styles.css">
+    <!-- Icon courtesy of Material Design https://material.io/resources/icons/?style=baseline -->
```
Comment (pavi2410) [inline index.html]: Remove

File: index.html [hunk 2]
```diff
@@ -6,6 +6,8 @@
...
+    <link rel="icon" type="image/svg" href="./assets/detective.png">
```
Comment (pavi2410) [inline index.html]:
```suggestion
    <link rel="shortcut icon" type="image/png" href="./assets/detective.png">
```

File: assets/location_searching-24px.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0z" fill="none"/><path d="M20.94 11c-.46-4.17-3.77-7.48-7.94-7.94V1h-2v2.06C6.83 3.52 3.52 6.83 3.06 11H1v2h2.06c.46 4.17 3.77 7.48 7.94 7.94V23h2v-2.06c4.17-.46 7.48-3.77 7.94-7.94H23v-2h-2.06zM12 19c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7z"/></svg>
```
Comment (pavi2410) [inline assets/location_searching-24px.svg]: Remove

File: assets/image_search-24px.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0V0z"/><path d="M18 13v7H4V6h5.02c.05-.71.22-1.38.48-2H4c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-5l-2-2zm-1.5 5h-11l2.75-3.53 1.96 2.36 2.75-3.54zm2.8-9.11c.44-.7.7-1.51.7-2.39C20 4.01 17.99 2 15.5 2S11 4.01 11 6.5s2.01 4.5 4.49 4.5c.88 0 1.7-.26 2.39-.7L21 13.42 22.42 12 19.3 8.89zM15.5 9C14.12 9 13 7.88 13 6.5S14.12 4 15.5 4 18 5.12 18 6.5 16.88 9 15.5 9z"/><path fill="none" d="M0 0h24v24H0z"/></svg>
```
Comment (pavi2410) [inline assets/image_search-24px.svg]: Remove

File: assets/search-24px.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/><path d="M0 0h24v24H0z" fill="none"/></svg>
```
Comment (pavi2410) [inline assets/search-24px.svg]: Remove

**Comment by JS-goose** [comment]
Requested changes made :)

**Comment by pavi2410** [comment]
Thanks! 😃

Closes #8

**Pull request merged**